ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,95 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  intermediate411:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.11"
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  stable:
```

Comment:
Sounds like rename it to `intermediate412` for easier  reading.

Reply:
Fixed